### PR TITLE
Rewrite help for `!init` commands.

### PR DESCRIPTION
**Pull request by Croebh**

### CODE REVIEW

**Comment by Croebh** [comment]
Rewrote most of the help for `!init` commands, so I renamed the PR.

File: cogs5e/initTracker.py [hunk 1]
```diff
@@ -89,14 +89,17 @@ def __init__(self, bot):
...
-                            -ac <AC> (sets combatant AC)
-                            -resist/immune/vuln <resistance>"""
+        
+        __Valid Arguments__
+        -h - Hides HP, AC, Resists, etc.
+        -p - Places combatant at the given modifier, instead of rolling
+        -controller <controller> - Pings a different person on turn.
+        -group <group> - Adds the combatant to a group.
+        -hp <hp> - Sets starting HP. Default: None.
+        -ac [ac] - Sets combatants AC. Default: None.
```
Comment (mommothazaz123) [inline cogs5e/initTracker.py]: should these be `<ac>` to signify they're required as above?

File: cogs5e/initTracker.py [hunk 2]
```diff
@@ -159,17 +162,18 @@ def __init__(self, bot):
...
-              -name [name scheme, use "#" for auto-numbering, ex. "Orc#"]
-              -h (same as !init add, default true)
-              -group (same as !init add)
-              -npr (removes physical resistances when added)
-              -rollhp (rolls monster HP)
-              -hp [starting hp]
-              -ac [starting ac]"""
+        __Valid Arguments__
+        adv/dis - Give advantage or disadvantage to the initiative roll.
+        -b [condition bonus] - Adds a bonus to the combatants nitiative roll.
```
Comment (mommothazaz123) [inline cogs5e/initTracker.py]: as above - [] vs <>
Also, nitiative

File: cogs5e/initTracker.py [hunk 3]
```diff
@@ -257,12 +261,13 @@ def __init__(self, bot):
...
         """Adds the current active character to combat. A character must be loaded through the SheetManager module first.
-        Args: adv/dis
-              -b [conditional bonus]
-              -phrase [flavor text]
-              -p [init value]
-              -h (same as !init add)
-              --group (same as !init add)"""
+        __Valid Arguments__ 
+        adv/dis - Give advantage or disadvantage to the initiative roll.
+        -b [condition bonus] - Adds a bonus to the combatants Initiative roll.
```
Comment (mommothazaz123) [inline cogs5e/initTracker.py]: See above

File: cogs5e/initTracker.py [hunk 4]
```diff
@@ -490,18 +499,17 @@ def __init__(self, bot):
...
-        -vuln <DMGTYPE>
-        -neutral <DMGTYPE>
-        -group <GROUP> (changes group)
-        -max <MAXHP> (sets max hp)
-        -hp <HP> (sets current hp)"""
+        -h - Hides HP, AC, Resists, etc.
+        -p <value> - Changes the combatants placement in the Initiative.
+        -name <name> - Changes the combatants name.
+        -controller <controller> - Pings a different person on turn.
+        -ac [ac] - Sets combatants AC.
```
Comment (mommothazaz123) [inline cogs5e/initTracker.py]: see above

File: cogs5e/initTracker.py [hunk 5]
```diff
@@ -490,18 +499,17 @@ def __init__(self, bot):
...
+        -resist [damage type] - Gives the combatant resistance to the given damage type.
+        -immune [damage type] - Gives the combatants immunity to the given damage type.
+        -vuln [damage type] - Gives the combatant vulnerability to the given damage type.
+        -neutral [damage type] - Removes the combatants immunity, resistance, or vulnerability to the given damage type.        -group <group> - Adds the combatant to a group. To remove them from group, use -group None.
```
Comment (mommothazaz123) [inline cogs5e/initTracker.py]: Missing newline

File: cogs5e/initTracker.py [hunk 6]
```diff
@@ -700,31 +708,32 @@ def __init__(self, bot):
...
-        -vuln [vulnability] - gives the combatant vulnerability
-        -neutral [neutral] - removes immune/resist/vuln
+        -resist [damage type] - Gives the combatant resistance to the given damage type.
+        -immune [damage type] - Gives the combatants immunity to the given damage type.
+        -vuln [damage type] - Gives the combatant vulnerability to the given damage type.`-custom` - Makes a custom attack with 0 to hit and base damage. Use `-b` and `-d` to add damage and to hit.
+        -neutral [damage type] - Removes the combatants immunity, resistance, or vulnerability to the given damage type.
         __General__
-        -ac [ac] - modifies ac temporarily; adds if starts with +/- or sets otherwise
-        -sb [save bonus] - Adds a bonus to saving throws"""
+        -ac [ac] - modifies ac temporarily; adds if starts with +/- or sets otherwise.
```
Comment (mommothazaz123) [inline cogs5e/initTracker.py]: as above - I think we should always use <> for tags that have arguments following

File: cogs5e/initTracker.py [hunk 7]
```diff
@@ -940,41 +1009,42 @@ def __init__(self, bot):
...
-        -i - Ignores Spellbook restrictions, for demonstrations or rituals.
+        -t [target] - Specifies one or more combatants to target, chainable (e.g., "-t or1 -t or2").
+        -i - Ignores Spellbook restrictions, for demonstrations or rituals. Doesn't use a spell slot.
         -l [level] - Specifies the level to cast the spell at.
         **__Save Spells__**
-        -dc [Save DC] - Default: Pulls a cvar called `dc`.
-        -save [Save type] - Default: The spell's default save.
-        -d [damage] - adds additional damage.
-        adv/dis - forces all saves to be at adv/dis.
+        -dc [save dc] - Changes the DC of the save. Default: Pulls a cvar called `dc`.
```
Comment (mommothazaz123) [inline cogs5e/initTracker.py]: We should undocument the default here, as that's no longer true

File: cogs5e/initTracker.py [hunk 8]
```diff
@@ -940,41 +1009,42 @@ def __init__(self, bot):
...
-        -dur [duration] - changes duration of spell effects.
-        int/wis/cha - different skill base for DC/AB"""
+        -phrase [phrase] - Adds flavor text.
+        -title [title] - Changes the title of the cast. Replaces [sname] with spell name.
+        -dur [duration] - Changes the duration of the spell effects, in rounds.
+        int/wis/cha - Uses a different ability score for spell DC and attack bonus."""
         return await self._cast(ctx, None, spell_name, args)
 
-    @init.command()
+    @init.command(aliases=['rc'])
```
Comment (mommothazaz123) [inline cogs5e/initTracker.py]: hehe, irc

File: cogs5e/initTracker.py [hunk 9]
```diff
@@ -1049,7 +1119,9 @@ def __init__(self, bot):
 
     @init.command()
     async def end(self, ctx, args=None):
-        """Ends combat in the channel."""
+        """Ends combat in the channel.
+        __Valid Arguments__
+        -force - Forces an init to end, in case its erroring."""
```
Comment (mommothazaz123) [inline cogs5e/initTracker.py]: it's

**Pull request merged**